### DISCUSSION TITLE
Can‘t open browser using integrated terminal

### DISCUSSION THREAD

**Discussion by cqh963852** [microsoft/WSL#6533] [General]
I am developing a react app, created by create-react-app.

If open the terminal in windows terminal,It can auto open the browser while start the app.

If open the terminal in vscode integerated terminal ,It won't open the browser,And it will open the git bash under windows. and give an error.
Besides I have had click an option after run the app in integerated terminal, but I can't reset it.(I tried reinstall vscode, genrate a new wsl instance,but not work)

![image](https://user-images.githubusercontent.com/17702287/105637695-a8d5cd00-5ea9-11eb-9896-e41c9daeee85.png)

Any one can help?

**Comment by benhillis** [comment]
Why are you using git bash?  That is a win32 applications and cannot run ELF binaries.

**Comment by cqh963852** [comment]
> Why are you using git bash? That is a win32 applications and cannot run ELF binaries.

The first I run the app in integrated terminal,The vscode ask me select a terminal to run another shell. I select git bash.
I don't know how to change it.

For now,If I run the app in integrated terminal,It will continue open git bash to run another shell(This shell may be open browser shell).
